City-of-Helsinki/helsinki-paatokset
# PP-338: Add a tag to motion to differentiate them from decisions.

**To test**
- Checkout branch and run `make drush-cr ahjo-migrations`
- Open some decisions: https://helsinki-paatokset.docker.so/fi/admin/content/decisions
  - Also filter by "Decision -> False" to find motions. Open a few of those too
- The motions should have a label on them that says "This is a motion", but none of the decisions should have the label
- The label should be the same as the "more recent handlings" label, and the following fixes should be applied:
  - The text should be bolded
  - There should be slightly more padding added to the right side 
- Edit a motion or a decision and toggle the "Node is a published decision" value
  - When the value is off, the label should appear
- Navigate using the dropdown. The label should appear and disappear dynamically if you select a decision after a motion
